packages/icons/package.json

```
@@ -4,13 +4,28 @@
    "main": "./index.tsx",
    "types": "./index.tsx",
    "license": "MIT",
+   "scripts": {
+      "clean": "",
+      "build": ""
+   },
    "peerDependencies": {
-      "@chakra-ui/react": ">= 1.8.3 < 2"
+      "@chakra-ui/react": ">= 1.8.3 < 2",
+      "@emotion/react": ">=11.0.0 <12.0.0",
+      "framer-motion": ">=3.0.0 <7.0.0",
+      "@emotion/styled": ">=11.0.0 <12.0.0",
+      "react-dom": ">=16.8.6 <17.0.0 || >=17.0.0 <18.0.0",
```

Similar to https://github.com/iFixit/react-commerce/pull/709#discussion_r982883122, we can probably simplify this requirement and the other `react-dom` requirement below.

```suggestion
      "react-dom": ">=16.8.6 <18.0.0",
```